lib/assets/gallery-video-article.mst

```diff
@@ -12,6 +12,9 @@
             <figure class="main-image">
                 <img data-libingester-asset-id="{{ main_image }}">
             </figure>
+            <div class="play-button-wrapper">
+                <img src="play-button.svg" />
```

Comment:
Hmm... I don't this path available in the built shard in this manner. I think you might need to post-process the file similarly to the quote image: https://github.com/endlessm/libingester/blob/4930408351bff245c808f2a7e2bfcdef4ecd644f/lib/index.js#L574-L581

@wgreenberg Any thoughts?